woof-code/rootfs-skeleton/etc/init.d/trim

```diff
@@ -0,0 +1,38 @@
+#!/bin/ash
+
+. /etc/rc.d/PUPSTATE
+[ $PUPMODE -eq 5 -o $PUPMODE -eq 6 ] && exit 0
+mountpoint -q /initrd/mnt/dev_save || exit 0
+
```

Comment:
should also check if is a rotating device interrogating /sys/block/sdX/queue/rotational or using lsblk.
There is no point to trim rotating disks I think

Reply:
It doesn't do anything, so it's safe to try `fstrim` even if it's not supported. And skipping `fstrim` after one failure is probably not the best idea.